packages/fa-icon-chooser/src/utils/utils.spec.ts

```diff
@@ -49,6 +49,43 @@ describe('parseSvgText', () => {
   it('tests duotone with only secondary', () => {
     expect(parseSvgText(duotoneSvgOnlySecondary)).toEqual([640, 512, [], null, ['M2 2 h2 z', '']]);
   });
+
+  describe('JSON format', () => {
```

Comment:
See my comment in `utils.ts` where `parseSvgText()` is defined. At the moment, it seems to me unnecessary and undesirable for `parseSvgText()` to try to handle parsing both SVG text and JSON text.